webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -108,13 +87,22 @@ class ManageManifestModal extends Component {
       isManifestImported,
       canEditOrganizations,
       simpleContentAccess,
+      enableSimpleContentAccess,
+      disableSimpleContentAccess,
+      taskInProgress,
+      manifestActionInProgress,
     } = this.props;
 
+    const actionInProgress = (taskInProgress || manifestActionInProgress);
```

Comment:
What's the difference between `taskInProgress` and `manifestActionInProgress`. High chance they're the same.

Reply:
The `taskInProgress` prop will have true value when we will get the success response of any manifest Action(I.e. Refresh, upload, delete, Enable/Disable SCA). While, when the action request is in progress `manifestActionInProgress` will have true value. So if we will get failure from any Manifest Action request, Only  `manifestActionInProgress` value will get change.